### DISCUSSION TITLE
Allow the name of the environment to be used as a variable

### DISCUSSION THREAD

**Discussion by Tarig0** [Kong/insomnia#3736] [Ideas]
- Insomnia Version: 7.0.3
- Operating System: Windows 10 x64

## Details
Small - QOLI
I have setup a few environments with a variable that is literally the name of sub environment.
maybe using the macro {% environment %}

For example {"baseUrl": "www{% environment %}.mydomain.com", "environment":"DevA"}

**Comment by welcome** [comment]
👋 Thanks for opening your first issue! If you're reporting a 🐞 bug, please make sure
you include steps to reproduce it. If you're requesting a feature 🎁, please provide real
use cases that would benefit. 👪

To help make this a smooth process, please be sure you have first read the
[contributing guidelines](https://github.com/getinsomnia/insomnia/blob/master/CONTRIBUTING.md).

**Comment by elemanhillary-zz** [comment]
Not detailed

**Comment by rafaelrenanpacheco** [comment]
This feature is way too specific and would make sense just for those with that exactly environment situation.

A simple, and already supported solution, is just do:

```json
{"baseUrl": "wwwDevA.mydomain.com"}
```

And change that for each environment.

**Comment by Tarig0** [comment]
This feature has been requested multiple times, so I don't think it is specific. I see this as an extension of the request => folder which is able to go as high as the workspace name.

**Comment by gschier** [comment]
I think this would be nice to have. We could have an environment tag with operations to do things on the current environment, one of them being to get the name

**Comment by Tarig0** [comment]
Do we have any other idea's on what could be done on the current environment?
